client/ayon_harmony/plugins/publish/validate_instances.py

```diff
@@ -34,27 +34,31 @@ def process(self, context, plugin):
             harmony.imprint(instance.data["setMembers"][0], data)
 
 
-class ValidateInstance(pyblish.api.InstancePlugin):
+class ValidateInstance(pyblish.api.InstancePlugin, OptionalPyblishPluginMixin):
```

Comment:
By the way, this validation may need to also check whether it matches the current task name? If you're up for that.

The maya, houdini and fusion validations e.g. I think do that - if you need a reference.